signinapp/team.py

```diff
@@ -1,12 +1,20 @@
 from flask import Blueprint, request
 from flask.templating import render_template
-from flask_login import login_required
+from flask_login import current_user, login_required
 
-from .model import Subteam
+from .model import Role, Subteam, User
 
 team = Blueprint("team", __name__)
 
 
+@team.route("/users")
+@login_required
```

Comment:
```suggestion
@mentor_required
```
Otherwise students could access this page if they found the URL.